### PR TITLE
Build for AlmaLinux 8

### PR DESCRIPTION
**Pull request by gwarf**
* Build and release for AlmaLinux 8
* Fix rpmlint call
* Powertools repository is needed on REHL8, and crb is only for RHEL9
* htcondor repository is needed, wlcg-repo do not contain real packages for el9, and nothing for el8
* Bump to 6.1.0

### CODE REVIEW

**Comment by gwarf** [comment]
@andrea-manzi please have a look, thanks!

**Comment by gwarf** [comment]
@andrea-manzi can you please review this PR?
We can let the RHLE8 package available here and let people add it to/take it from whatever repo they may be using.
Or we can just close/cancel this PR.

Same as for https://github.com/EGI-Federation/wn-metapackage/pull/4#issuecomment-2167584175

File: CHANGELOG [hunk 1]
```diff
@@ -8,8 +8,12 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [6.1.0]
+- Build and package for RHEL8 (#11) (Baptiste Grenier)
+
 ## [6.0.1]
 - Add ARC REST, ginfo and lcg-infosites on RHEL9 (#10) (Baptiste Grenier)
+- Build and pacakge for RHEL8 (#11) (Baptiste Grenier)
```
Comment (andrea-manzi) [inline CHANGELOG]:
```suggestion
```

**Review by andrea-manzi — APPROVED** [review]
LGTM! just a small change in the CHANGELOG

**Pull request merged**